### ISSUE TITLE
Admin Panel seems disappers With 0.13.4

### ISSUE THREAD

**Issue by corasaniti** [alexjustesen/speedtest-tracker#931]
With version 0.13.4 When go in result page and then return in Home page, the Admin Panel seems disappers.
I'm forced to press the "Admin Panel" button again
Thanks

**Comment by alexjustesen** [comment]
Has it happened again or just one? Also can you open the container cli and run `php artisan optimize:clear` and let me know if the issue continues?

**Comment by alexjustesen** [comment]
Linking to #930 as they might be related.

**Comment by corasaniti** [comment]
@alexjustesen 
I ran the following command but it didn't work. 
`php artisan optimize:clear`
The admin panel keeps disappearing
Thank you
![Screenshot 2023-11-27 141419](https://github.com/alexjustesen/speedtest-tracker/assets/5001932/bf4e9be5-c49e-48be-8a94-3f19868bbd93)

**Comment by alexjustesen** [comment]
@corasaniti pretty sure this is linked to #930 so going to close this as a dup and track the fix there.

**Comment by alexjustesen** [comment]
@corasaniti after re-reading this "Home" and "Dashboard" are different pages. "Home" brings you to the front end that doesn't require auth and "Dashboard" is the admin panel's dashboard.

**Comment by corasaniti** [comment]
@alexjustesen 
Yes you are right. If I go back to the Dashboard from the Result page, the Admin Panel remains visible. 
The behavior is correct. I'm closing the issue
Thank you
